[2025-04-17T12:42:09.391+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-17T12:42:09.617+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: beer_data_processing.process_robust_scaler manual__2025-04-17T12:39:29.171572+00:00 [queued]>
[2025-04-17T12:42:09.720+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: beer_data_processing.process_robust_scaler manual__2025-04-17T12:39:29.171572+00:00 [queued]>
[2025-04-17T12:42:09.725+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-17T12:42:09.865+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_robust_scaler> on 2025-04-17 12:39:29.171572+00:00
[2025-04-17T12:42:10.022+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6476) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-17T12:42:10.096+0000] {standard_task_runner.py:72} INFO - Started process 6626 to run task
[2025-04-17T12:42:10.181+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'beer_data_processing', 'process_robust_scaler', 'manual__2025-04-17T12:39:29.171572+00:00', '--job-id', '1021', '--raw', '--subdir', 'DAGS_FOLDER/beer_data_processing_dag.py', '--cfg-path', '/tmp/tmpq2jjyr8u']
[2025-04-17T12:42:10.192+0000] {standard_task_runner.py:105} INFO - Job 1021: Subtask process_robust_scaler
[2025-04-17T12:42:12.916+0000] {task_command.py:467} INFO - Running <TaskInstance: beer_data_processing.process_robust_scaler manual__2025-04-17T12:39:29.171572+00:00 [running]> on host a0d0456bae2c
[2025-04-17T12:42:13.708+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='beer_data_processing' AIRFLOW_CTX_TASK_ID='process_robust_scaler' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T12:39:29.171572+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-17T12:39:29.171572+00:00'
[2025-04-17T12:42:13.715+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-17T12:42:13.717+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-17T12:42:13.719+0000] {logging_mixin.py:190} INFO - Current task name:process_robust_scaler state:running start_date:2025-04-17 12:42:09.621744+00:00
[2025-04-17T12:42:13.727+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing and current dag run status:running
[2025-04-17T12:42:13.732+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-17T12:42:14.409+0000] {logging_mixin.py:190} INFO - Robust Scaling Example - Original: {'ABV': 10.0, 'Min IBU': 60.0, 'Max IBU': 100.0, 'number_of_reviews': 218.0}
[2025-04-17T12:42:14.413+0000] {logging_mixin.py:190} INFO - Robust Scaling Example - Scaled: {'ABV': 1.5384615384615388, 'Min IBU': 4.0, 'Max IBU': 3.25, 'number_of_reviews': 0.4789272030651341}
[2025-04-17T12:42:14.715+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-04-17T12:42:14.880+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-17T12:42:14.882+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=beer_data_processing, task_id=process_robust_scaler, run_id=manual__2025-04-17T12:39:29.171572+00:00, execution_date=20250417T123929, start_date=20250417T124209, end_date=20250417T124214
[2025-04-17T12:42:14.994+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-17T12:42:14.996+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-17T12:42:14.997+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing queued_at:2025-04-17 12:39:29.407696+00:00
[2025-04-17T12:42:14.998+0000] {logging_mixin.py:190} INFO - Task hostname:a0d0456bae2c operator:PythonOperator
[2025-04-17T12:42:15.071+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-17T12:42:15.178+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-17T12:42:15.189+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-17T12:42:15.196+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
